libs/features/mas/src/variants/plans-edu.js

```diff
@@ -0,0 +1,50 @@
+import { Plans } from './plans.js';
+import { CSS } from './plans-edu.css.js';
+import { css } from 'lit';
+
+export const PLANS_EDU_AEM_FRAGMENT_MAPPING = {
```

Comment:
is mapping different for plans edu? just wondering if we could reuse plans mapping here

Reply:
Nobody requested different mapping. In my opinion it is better if we don't show in card editor for "Plans students" component that are not needed for that card. But I already refactored the code and will use the same mapping. Let's keep it simple.